TTBot/Models/ChampionshipResultsModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using ServiceStack.DataAnnotations;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TTBot.Models
+{
+    [Alias("ChampionshipResults")]
+    public class ChampionshipResultsModel
+    {
+        [AutoIncrement]
+        [PrimaryKey]
+        public int Id { get; set; }
+        public string Driver { get; set; }
+
+        [ForeignKey(typeof(Event))]
+        public int EventId { get; set; }
+        public string[] Positions { get; set; }
```

Comment:
Does this go in db as a json array?

Reply:
I'm not sure how it goes in to the DB. I'll take a look and get back to you.

Reply:
It is stored as an NVARCHAR

![image](https://user-images.githubusercontent.com/19607943/105607369-1f81a600-5d96-11eb-90bf-899a01f0d947.png)

Reply:
Data looks like this:

`Positions
[1*,5,2,1,2]
[2,3,3,2,3]
[3,4,4,5*,4]
[4,1,5,3,1*]
[5,2,1*,4,5]
[1*,2,1,2*]
[2,1*,3,3]
[3,3,1*,1]
`

Reply:
Data looks like this:

![image](https://user-images.githubusercontent.com/19607943/105607403-50fa7180-5d96-11eb-9b88-a6fb6eba114c.png)